nimble/transport/nrf5340/src/nrf5340_ble_hci.c

```diff
@@ -19,11 +19,13 @@
 
 #include <assert.h>
 #include <string.h>
+#include <syscfg/syscfg.h>
 #include <sysinit/sysinit.h>
 #include <nimble/ble.h>
 #include <ipc_nrf5340/ipc_nrf5340.h>
 #include <nimble/transport.h>
 #include <nimble/transport/hci_ipc.h>
+#include <nimble/transport/transport_ipc.h>
```

Comment:
this is included from nimble/transport if hci_ipc package is included in build and it should be, because it's added as dependency to nrf5340 transport package - you likely do not have recent newt which generates required defines in syscfg.h